tests-utils/src/main/scala/tech/beshu/ror/utils/containers/windows/WindowsEsSetup.scala

```diff
@@ -107,4 +111,28 @@ object WindowsEsSetup extends LazyLogging {
       val updated = file.lines.map(line => if (line.startsWith(prefix)) newLine else line)
       file.overwrite(updated.mkString("\n"))
 
+  private def withRetries(maxRetries: Int, cleanBeforeRetrying: => Unit)(block: => Unit): Unit = {
```

Comment:
we have retry-like utils in:
1. ScalaOps.scala
2. ScalaUtils.scala
3. EsStartupChecker.scala
4. here

Maybe we can refactor it a little bit and use ScalaOps in all the places?
Or at least have utils for plugin code and test code? WDYT?